src/miner.h

```diff
@@ -26,6 +26,16 @@ static const bool DEFAULT_PRINTPRIORITY = false;
 
 struct CBlockTemplate
 {
+    CBlockTemplate(const CBlockTemplate &tmpl) :
+        block(tmpl.block), vTxFees(tmpl.vTxFees),
+        vTxSigOpsCost(tmpl.vTxSigOpsCost),
+        vchCoinbaseCommitment(tmpl.vchCoinbaseCommitment)
+    {
+        block.vtx = tmpl.block.vtx;
+    }
```

Comment:
Apparently it doesn't (I was confused by whether a CBlock could be copy-constructed from another CBlock).  Will remove.